docs/starters.yml

```diff
@@ -796,3 +796,20 @@
     - Based on default starter
     - Bulma Css
     - Sass based Styling
+- url: https://i18n.smakosh.com/
+  repo: https://github.com/smakosh/gatsby-starter-i18n
+  description: Gatsby v2 Starter with i18n using react-intl and more cool features.
+  tags:
+    - Styling:Styled-Components
+    - Recompose
+    - i18n
+    - react-intl
+    - Formik
+    - Yup
+    - Netlify Form
+  features:
+    - Based on default starter
+    - i18n with change of direction
```

Comment:
Could you expand upon this maybe? I'm presuming it means rtl text, but that's not immediately clear!

Reply:
yes that's what it means, sorry I'm bad with words 😄